src/NServiceBus.AcceptanceTests/Sagas/When_adding_state_to_context.cs

```diff
@@ -14,7 +14,7 @@ namespace NServiceBus.AcceptanceTests.Sagas
     [TestFixture]
     public class When_adding_state_to_context : NServiceBusAcceptanceTest
     {
-        [Test]
+        [Test, Ignore("Not sure what to do here, since the start message has no corr prop we can't generate the SagaId, perhaps custom finder isn't supported for the devstorage?")]
```

Comment:
Moved the test to the core folder and made it use the InMemory persister for now